cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -42,6 +42,8 @@ Replacement syntax for deprecated and removed features are also indicated.
 | `0...`                      | Syntax        | Deprecated   | Replaced by `0o...` (see above).
 | `0X...`                     | Syntax        | Deprecated   | Only `0x...` (lowercase x) will be supported.
 | `SET [PLAINTEXT \| ENCRYPTED] PASSWORD` | Syntax | Added | For `CREATE USER` and `ALTER USER` it is now possible to set (or update) a password when the plaintext password is unknown, but the encrypted password is available.
+| `SHOW CURRENT USER`      | Syntax    | Added      | New Cypher command for showing current user, replacing the procedure
```

Comment:
I think this line stops abruptly. I'd prefer to remove the part about replacing the procedure.